### ISSUE TITLE
New user rights

### ISSUE THREAD

**Issue by phanosh** [luethi/Discharge_Optical_Measurement#4]
when creating a new user profile, one is automatically part of the "Open World" organization. There is an issue because all new users are able to measure on all sites. This may create a big problem if random people take false measurements and add the to the database. 

I think the ability to measure on sites should be removed from the default rights of a new user, and should only exist once a user is added to a new organization.

cheers, 
Phanos

**Comment by luethi** [comment]
its a good point and I am just trying to reproduce it... .Stay tuned....

**Comment by luethi** [comment]
I will discuss it with Lovre. There is a little bug somewhere, ....

**Comment by luethi** [comment]
ok: I talked to Lovre. The background is that I requested some contradictory things. Now Encoders have green light to do as they have wanted to do.
In the future, it will be something like this:
A new user is member of open world, but some guy with the right permissions needs to ADD him to particular organizations (and its suporganizations). So a new user wil only be able to look and take measurements if he is explicitly added to the group.
Thanks for this comment, it helped me clear up some things.
I let you close the ticket, if it is ok.
